### ISSUE TITLE
Compatibility with Windows 10 dark mode.

### ISSUE THREAD

**Issue by bash550** [BeipDev/BeipMU#6]
Would it be possible to make the client compatible with this mode in order to turn the borders black?
Something like the way MS Edge looks when using the windows native dark theme. With black drop down menus and white text.

**Comment by BeipDev** [comment]
After seeing that other apps like 'Notepad' don't get the dark theme, it looks like only apps using a diffeent UI system get the dark theme. If you set the OS to high contrast mode BeipMU will appear dark.

This doesn't really solve your problem, but even Windows itself isn't consistent here.

**Comment by bash550** [comment]
Quite true. Ah well, we can manage it as it is. Thanks!

**Comment by AxelTerizaki** [comment]
I know Windows is an idiot when it comes to managing its dark mode but I wouldn't mind if this issue would be reopened and looked at. I used the high contrast mode given I'm nearly blind and light stuff tends to blind me, but I'd like to switch to Windows' dark theme since it looks much better. Alas, BeipMu isn't using it :/

Thanks in advance.

**Comment by BeipDev** [comment]
Wow, this is a pretty old issue now. I actually started adding a dark mode in the latest builds, look in Preferences for 'Dark Mode', it'll make everything but the title/menu dark.

There was another user with similar visual complaints and these changes made them happier. I eventually want to remove the menu bar entirely in favor of a burger menu style icon, as that can make things even darker.

**Comment by AxelTerizaki** [comment]
Indeed, I never saw that option. Looks like I should read changelogs more often :)

It's not perfect though, as you said, the menu and dialog boxes (including the options one) aren't affected. Would there be an easy way for you to fix this sometime?

**Comment by AxelTerizaki** [comment]
I'll add that it's not a big, big, big issue actually. After fumbling with other apps who weren't using the dark mode I realized my quest was vain and that even some Windows apps or dialog boxes are inconsistent with it and barely use it, I gave up and switched back to the ugly high contrast theme again.

Sorry for bothering you with this ^^

**Comment by BeipDev** [comment]
Yeah, unfortunately the OS "Dark Mode" is unconnected to old style Win32 apps like BeipMU, only new UWP XAML ones pick it up. The dark mode stuff I have in there now is done by my own windows and some hacks for things like the scrollbars.

It's not perfect, but at least the dialogs are temporary, and most of the time it can be dark.

If I find an easy way I'd do it! I tried a bunch of things to make the menu bar dark but none worked. The OS handles the menu bar, so unless I make my own menu bar, I'm stuck there sadly.

Not bothering at all! The more people ask about something the more I think about it.

**Comment by BeipDev** [comment]
Randomly, do you use a screen reader?

**Comment by AxelTerizaki** [comment]
No, I don't :) I'm looking at the screen from close range since I can still see a bit. My main issue with apps is if they use light elements in their interfaces.

Some, for example, obey partially to Windows' High Contrast mode, and use a white, forced background, but don't override the font color so it results in white font on white background. Fortunately these kind of apps are less and less common but they still do exist.

**Comment by BeipDev** [comment]
Yeah, hard to test on all of the color combinations out there. I prefer dark mode things in general these days. If you have any ideas let me know.

**Comment by AxelTerizaki** [comment]
I know this has been a long time, but I don't suppose there are any plans to support dark mode anytime soon since this is a win32 app? I'm trying to adjust my win 11 install to dark mode instead of high contrast mode (because the latter is way too ugly) but all window dialogs are still in light mode.

Also out of curiosity are there any docs/how tos anywhere to make Beipmu work on Linux via Wine? I've tried a bit a few months back and it was a real mess and I couldn't make it work properly/easily.

**Comment by BeipDev** [comment]
I did improve the darkness by removing the bright menu bar, but I haven't got any ideas on the dialogs. The OS handles those and anything I do to make them dark is not complete. So no solution there until I write my own UI library...

To run on Wine, all I had to do was just install the latest Wine staging, then it just ran! (unzipping it to a folder and just running the .exe) Nothing fancy. The tricky part was just installing Wine itself. Only the most recent versions work nicely, like 8.x+
